picard/ui/tablebaseddialog.py

```diff
@@ -91,12 +95,27 @@ def resizeEvent(self, event):
 
 class SortableTableWidgetItem(QtWidgets.QTableWidgetItem):
 
-    def __init__(self, sort_key):
+    def __init__(self, obj=None):
         super().__init__()
-        self.sort_key = sort_key
+        self._obj = obj
+
+    def setText(self, text):
+        return super().setText(text)
```

Comment:
What's the point of overwriting `setText` without any change here?

Reply:
Just a leftover from previous experiment, removed.